### PR TITLE
[Chore] Update changelog for hotfixes

### PR DESCRIPTION
**Pull request by sc-addypathania**
Update changelog for 21.0.10, 21.1.3 21.1.4 21.1.5

- [x] This PR follows the [Contribution Guide](https://github.com/Sitecore/jss/blob/dev/CONTRIBUTING.md)
- [x] Changelog updated

## Testing Details

- [ ] Unit Test Added
- [ ] Manual Test/Other (Please elaborate)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -56,6 +56,30 @@ Our versioning strategy is as follows:
...
+
+* `[create-sitecore-jss]` Change ^ to ~ for versioning in templates.
+
+## 21.1.3
+
+### 🐛 Bug Fixes
+
+*  Fix installed sitecore-jss-* dependency version. Change ^ to ~ ([#1481](https://github.com/Sitecore/jss/pull/1481))
+
+## 21.0.10
```
Comment (illiakovalenko) [inline CHANGELOG.md:77]: This version should be below 21.1.0

**Review by illiakovalenko — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -56,6 +56,30 @@ Our versioning strategy is as follows:
...
+
+### 🐛 Bug Fixes
+
+* `[create-sitecore-jss]` Change ^ to ~ for versioning in templates.
+
+## 21.1.3
+
+### 🐛 Bug Fixes
+
+*  Fix installed sitecore-jss-* dependency version. Change ^ to ~ ([#1481](https://github.com/Sitecore/jss/pull/1481))
```
Comment (illiakovalenko) [inline CHANGELOG.md:75]:
```suggestion
* Fix installed sitecore-jss-* dependency version. Change ^ to ~ ([#1481](https://github.com/Sitecore/jss/pull/1481))
```

File: CHANGELOG.md (hunk 1 above)
Comment (sc-addypathania) [reply CHANGELOG.md:77]: it should be 21.0.10 but should be below 21.1.x

File: CHANGELOG.md [hunk 3]
```diff
@@ -56,6 +56,30 @@ Our versioning strategy is as follows:
...
+
+## 21.0.10
+
+### 🐛 Bug Fixes
+
+*  Fix installed sitecore-jss-* dependency version. Change ^ to ~ ([#1481](https://github.com/Sitecore/jss/pull/1481))
```
Comment (illiakovalenko) [inline CHANGELOG.md:81]:
```suggestion
* Fix installed sitecore-jss-* dependency version. Change ^ to ~ ([#1481](https://github.com/Sitecore/jss/pull/1481))
```

File: CHANGELOG.md (hunk 1 above)
Comment (illiakovalenko) [reply CHANGELOG.md:77]: yeah, so needs to be moved below

File: CHANGELOG.md [hunk 4]
```diff
@@ -142,6 +160,18 @@ Our versioning strategy is as follows:
 
 * `[sitecore-jss-angular]` Due to the Angular version upgrade and the change in _sitecore-jss-angular_ package output format  _sitecore-jss_ exports are not available in angular app scripts (src/scripts) via '@sitecore-jss/sitecore-jss-angular'. Please use '@sitecore-jss/sitecore-jss-angular/cjs' import instead. Check bootstrap.ts scripts as for a usage example.
 
+## 21.0.10
+
+### 🐛 Bug Fixes
+
+* Change ^ to ~ in sitecore-jss templates
+
+## 21.0.10
```
Comment (illiakovalenko) [inline CHANGELOG.md:169]:
```suggestion
## 21.0.9
```

**Review by illiakovalenko — APPROVED** [review]

**Pull request merged**